src/numus.c

```diff
@@ -0,0 +1,69 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <string.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#define MAX_NODES 1024 /* kernel max */
```

Comment:
Maybe it would be a better idea to use CONFIG_NODES_SHIFT or some other #def ?
What about [this](https://docs.huihoo.com/doxygen/linux/kernel/3.7/include_2linux_2numa_8h_source.html) code?